### ISSUE TITLE
Flame, Fire Aspect enchant and lava

### ISSUE THREAD

**Issue by Sikatsu** [Rocologo/MobHunting#353]
Hey,

Killing mobs with the Flame or Fire Aspect enchant, or hitting a mob into the lava will not give the player any money. It should automatically go into their balance. (so no item drop)

**Comment by Rocologo** [comment]
I guess I have to learn how to make these enchants???? 

And yes pushing a mob into lava should drop a reward, except the reward will be dropped where the mob dies... in lava....

**Comment by Sikatsu** [comment]
Point is that I don't use item drop, so shouldn't it go into the balance then?

What do you mean with these enchants?

**Comment by Rocologo** [comment]
I dont know how kill the mob with flame enchant.

**Comment by Sikatsu** [comment]
Oh

**Comment by Rocologo** [comment]
Can you explain to me how I make a flame enchant?

**Comment by Sikatsu** [comment]
Just like any other enchant? You can get it from villagers.

You could also use essentials:

/i dsword 1 flame:1 fireaspect:1

**Comment by Rocologo** [comment]
Perfect thank you :-) Ill use the essentials command for testing

**Comment by Rocologo** [comment]
I get an error
file:///home/christian/.minecraft/screenshots/2018-02-27_20.02.19.png

**Comment by Rocologo** [comment]
I get this error???
![2018-02-27_20 02 19](https://user-images.githubusercontent.com/1008133/36748848-61431f6a-1bf9-11e8-8b48-bb212ac7fd39.png)

**Comment by Sikatsu** [comment]
Use it on a bow, not on an arrow.

**Comment by Rocologo** [comment]
I still get the same error ???

**Comment by Rocologo** [comment]
"/enchant flame 1" did work 

But i got money from the killing

**Comment by Sikatsu** [comment]
/i bow 1 flame:1

Or /enchant flame 1 yes.

According to players they don't receive money using these enchants.

**Comment by Rocologo** [comment]
I did only test the bow/flame enchantment yesterday, and I saw that the money was dropped next to the player (and not where the mob died). And the money was immediately picked up. Maybe your players didnt notice this? I checked the console and made sure that the money was dropped. 

I have made a new snapshot where the money is dropped where the mob dies. But I will test a bit more when I get back - home from work.
